Assets/Scripts/PlayerInventory.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerInventory : MonoBehaviour
+{
+    Hashtable inventory;
+    // Start is called before the first frame update
+    void Start()
+    {
+        inventory = new Hashtable();
+    }
+    
+    public void addToInventory(string name, int amt) {
```

Comment:
below, it looks like `amt` can be a string to allow for 'all'. This should also accept a string and validate that the string is an integer

Reply:
Or have `removeFromInventory` only accept `int` for `amt` and have a coded otherwise unused value (maybe -1) indicate to remove all

Reply:
I like the idea of using "-1" much more than parsing ints everywhere.